tools/test.py

```diff
@@ -255,6 +256,9 @@ def HasRun(self, output):
         logger.info('#' + l)
       for l in output.output.stdout.splitlines():
         logger.info('#' + l)
+    elif output.HasSkipped():
+      skip = is_skipped.findall(output.output.stdout)      
```

Comment:
@bnoordhuis +1. It is Pythonic :-) I would still compile the RegEx with `re.compile`.